index.js

```diff
@@ -22,36 +22,50 @@ async function promise(cmd) {
 				handleError(error);
 				reject();
 			}
-			resolve();
+			const returnValue = { stdout, stderr };
+			resolve(returnValue);
 		});
 	});
 }
 
 module.exports = async (opt = {}) => {
 	const defaultOptions = {
 		path: ``,
-		cmd: ``
+		cmd: ``,
+		returnStdout: false,
+		returnStderr: false,
 	};
 
 	const options = { ...defaultOptions, ...opt };
 
-	const { path, cmd } = options;
+	const { path, cmd, returnStderr, returnStdout } = options;
+
+	let returnValue = {
+		stdout: [],
+		stderr: [],
+	};
 
 	// changes directory if a path exists
 	path !== `` ? process.chdir(path) : null;
 
 	// checks if there is one command or array of commands
 	if (typeof cmd !== 'object') {
-		return promise(cmd);
+		const output = await promise(cmd);
+		returnValue.stderr = returnStderr ? output.stderr : [];
+		returnValue.stdout = returnStdout ? output.stdout : [];
+		return Promise.resolve(returnValue);
 	} else {
 		for (let i = 0; i < cmd.length; i++) {
-			await promise(cmd[i]);
-
+			const output = await promise(cmd[i]);
+			if (returnStderr) {
+				returnValue.stderr.push(output.stderr);
+			}
+			if (returnStdout) {
+				returnValue.stdout.push(output.stdout);
+			}
 			let j = i + 1;
 			if (j === cmd.length) {
-				return new Promise((resolve, reject) => {
-					resolve();
-				});
+				return Promise.resolve(returnValue);
```

Comment:
@jeffreybradley1963 I am not sure if it is going to return a new Promise with "new" keyword. Can you instead send the `returnValue` inside the resolve like

```js
return new Promise((resolve, reject) => {
	resolve(returnValue);
});
```